src/SocialIcon/SocialIcon.tsx

```diff
@@ -165,24 +163,37 @@ export const SocialIcon: RneFunctionComponent<SocialIconProps> = ({
   loading,
   onLongPress,
   onPress,
-  Component = onPress || onLongPress ? TouchableHighlight : View,
+  onPressOut,
+  onPressIn,
+  Component = onPress || onLongPress || onPressIn || onPressOut
```

Comment:
Please deal with this case in `website/scripts/docgenParser.ts` @arpitBhalla

Reply:
How?

Reply:
Add conditions, as I have added for other props.